src/Pages.jsx/ViewAll.jsx

```diff
@@ -0,0 +1,38 @@
+import ArtHeader from "../Components/ArtHeader"
+import ViewAllArticles from "../Components/ViewAllArticles"
+import {useState, useEffect} from "react"
+import { getArticles } from "../Utils/Queries"
+
+
+const ViewAll = () => {
+    const[articles, setArticles] = useState([])
+    const[isLoading, setIsLoading] = useState(true)
+
+
+
+    useEffect(() => { 
+        getArticles()
+            .then((res) => { 
+            setArticles(res)
+            setIsLoading(false)
+            })
+        }, [])
+    
+
+        if (isLoading) { 
+    return <>
+    <h2>Loading...</h2>
+    </>
+    } 
+    else 
+    { 
+    return ( 
+        <> 
+        <ArtHeader /> 
+        <ViewAllArticles articles = {articles} /> 
```

Comment:
how can a user navigate to the single article page? right now the only way I could do this as a user was to enter the id manually in the url. Again, have a look at the react router NC notes/react router dom docs!